utils/scripts/install-el8.sh

```diff
@@ -7,8 +7,9 @@
 # interactively then these two commands can be used to set dnf into automatic mode.
 # dnf --assumeyes install dnf-plugins-core
 # dnf config-manager --save --setopt=assumeyes=True
+set -e
```

Comment:
Not needed, it's done on the #! line.

Reply:
A big part of the Dockerfile (and helper scripts) changes were to bring them into sync with each other so that if you open them all, for example with vim -d the only differences between them are the things that truly need to be different due to O/S differences, etc.

sudo was added here because it was present in another install-*.sh. IIRC the `centos7` one.

Also worth noting that the shebang arguments are ignored if a script is sourced rather than executed, so this takes care of that case.